monkey/monkey_island/cc/services/reporting/report.py

```diff
@@ -102,39 +112,44 @@ def get_tunnels():
     def get_scanned():
         formatted_nodes = []
 
-        nodes = ReportService.get_all_displayed_nodes()
-
-        for node in nodes:
-            # This information should be evident from the map, not sure a table/list is a good way
-            # to display it anyways
-            nodes_that_can_access_current_node = node["accessible_from_nodes_hostnames"]
-            formatted_nodes.append(
-                {
-                    "label": node["label"],
-                    "ip_addresses": node["ip_addresses"],
-                    "accessible_from_nodes": nodes_that_can_access_current_node,
-                    "services": node["services"],
-                    "domain_name": node["domain_name"],
-                    "pba_results": node["pba_results"] if "pba_results" in node else "None",
-                }
-            )
+        machines = ReportService._machine_repository.get_machines()
 
-        logger.info("Scanned nodes generated for reporting")
+        for machine in machines:
+            accessible_from = ReportService.get_scanners_of_machine(machine)
+            if accessible_from:
+                formatted_nodes.append(
+                    {
+                        "hostname": machine.hostname,
+                        "ip_addresses": list(map(str, machine.network_interfaces)),
```

Comment:
Should this be using the IPs and not the network interfaces? Or should we change the key to `network_interfaces`?